src/state.js

```diff
@@ -187,8 +187,12 @@ const removeAllPublishers = () => {
  */
 const addSubscriber = (subscriber) => {
   const type = subscriber.stream.videoType;
```

Comment:
We can just import `pathOr`:
`const { pathOr } = require('./util');`

and then just change one line:
`const type = pathOr('sip', 'stream.videoType', subscriber);`